core/platform/auth/firebase_auth_services.py

```diff
@@ -175,10 +160,11 @@ def mark_user_for_deletion(user_id):
         assoc_by_auth_id_model.put()
 
     try:
-        with _acquire_firebase_context():
+        with _firebase_admin_context():
             firebase_auth.update_user(assoc_by_auth_id_model.id, disabled=True)
-    except (ValueError, firebase_exceptions.FirebaseError) as e:
-        logging.exception(e)
+    except (firebase_exceptions.FirebaseError, ValueError):
```

Comment:
Why is ValueError a possibility?

Reply:
Is that an expected error? If not, we should not shadow it.

Reply:
Actually, same thing for the other cases -- are these expected errors?

Reply:
No, but I don't want these errors to crash the wipeout jobs. Logging an exception and reporting an error allows the job to complete.

Reply:
OK, please add a comment to explain the rationale for not raising.

Can you also check explicitly with @vojtechjelinek whether this is the way to go?